sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -211,7 +217,9 @@ export interface RecognizeLinkedEntitiesSuccessResult extends TextAnalyticsSucce
 export type RecognizePiiEntitiesErrorResult = TextAnalyticsErrorResult;
 
 // @public
-export type RecognizePiiEntitiesOptions = TextAnalyticsOperationOptions;
+export interface RecognizePiiEntitiesOptions extends TextAnalyticsOperationOptions {
+    domainFilter?: PIIEntityDomain;
```

Comment:
PiiEntityDomainType?

Reply:
we discussed the name of the type in this [thread](https://github.com/Azure/azure-sdk-for-js/pull/11015#discussion_r483276249).